core/src/main/scala/caliban/transformers/Transformer.scala

```diff
@@ -314,7 +314,7 @@ object Transformer {
           step.copy(fields =
             fieldName =>
               if (inner.contains(fieldName)) {
-                val args = field.fieldType.allFieldsMap(fieldName).allArgNames
+                val args = field.fieldType.getFieldOrNull(fieldName).allArgNames
```

Comment:
Seems unsafe?

Reply:
AFAICT it shouldn't be possible to get here without the field being present on the `__Type`, but I guess it's better to be safe than sorry. I'll add a check